ergo/logistic.py

```diff
@@ -34,8 +34,6 @@ class LogisticMixtureParams():
 def fit_single_scipy(samples) -> LogisticParams:
     with onp.errstate(all='raise'):  # type: ignore
         loc, scale = oscipy.stats.logistic.fit(samples)
-        scale = min(max(scale, 0.02), 10)
```

Comment:
I moved the weird Metaculus-specific clipping of loc and scale, as well as figuring out the low and high params, all to metaculus.py

Maybe there's an argument for doing some of all of it in logistic.py, but I think it'd be nice if logistic.py is just about logistics in general and doesn't care about weird Metaculus stuff